appinfo/app.php

```diff
@@ -71,6 +72,22 @@
 }
 
 $redirectSituation = false;
+
+$user = $userSession->getUser();
```

Comment:
this block could go to Appinfo\Application :) since there's enough stuff lying around here it can be moved around with a subsequent PR

Reply:
for now I just want to get it fixed and don't start a general cleanup round. We need to restructure this anyway completely from scratch once we allow multiple idp's and user back-ends